x-pack/test/security_solution_api_integration/config/serverless/config.base.ts

```diff
@@ -11,15 +11,18 @@ export interface CreateTestConfigOptions {
   kbnTestServerArgs?: string[];
   kbnTestServerEnv?: Record<string, string>;
 }
+import { services } from '../../../../test_serverless/api_integration/services';
 
 export function createTestConfig(options: CreateTestConfigOptions) {
   return async ({ readConfigFile }: FtrConfigProviderContext) => {
     const svlSharedConfig = await readConfigFile(
       require.resolve('../../../../test_serverless/shared/config.base.ts')
     );
-
     return {
       ...svlSharedConfig.getAll(),
+      services: {
```

Comment:
Isn't it the same as just `services,`?

Reply:
Yes true, I added it in case we want to pass additional services for the Security